### DISCUSSION TITLE
automatically fix linting errors/warnings?

### DISCUSSION THREAD

**Discussion by khanna7** [khanna-lab/cadre#44] [Q&A] [answered]
Have any of you used autopep8 (or something similar) to automatically fix errors/warning caught by flake8? Would you recommend using autopep8 (or something similar to it)?

**Comment by ncollier** [comment, marked as answer]
I have not. Might be useful if you can run it on bits code, rather than it automatically start formatting thing for you.

**Comment by khanna7** [comment]
Sounds good, thank you so much! I’ll try it in a separate branch, and on small code pieces. 
Aditya
